src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31104,38 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040000 "2023-04-03T14:54:32Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10719353") oboInOwl:hasBroadSynonym obo:CL_4040000 "glial restricted precursor")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11921204") oboInOwl:hasExactSynonym obo:CL_4040000 "tripotential glial-restricted precursor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4040000 "GRP cell")
+AnnotationAssertion(rdfs:label obo:CL_4040000 "glial restricted tripotential precursor cell")
+SubClassOf(obo:CL_4040000 obo:CL_0000123)
+SubClassOf(obo:CL_4040000 obo:CL_0011115)
+
+# Class: obo:CL_4040001 (chorionic girdle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") Annotation(oboInOwl:hasDbXref "PMID:35328607") obo:IAO_0000115 obo:CL_4040001 "Horse-specific highly invasive trophoblast cell that invades the endometrium where it forms endometrial cups.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040001 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040001 "2023-04-03T15:08:05Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") oboInOwl:note obo:CL_4040001 "Humans, mice and rats have a highly invasive (hemochorial) placentation. Horses have a non-invasive epitheliochorial placenta, except for the transient endometrial cup.")
+AnnotationAssertion(rdfs:label obo:CL_4040001 "chorionic girdle cell")
+SubClassOf(obo:CL_4040001 obo:CL_0000351)
+SubClassOf(obo:CL_4040001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003124))
+
+# Class: obo:CL_4040002 (enteroglial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16336493") Annotation(oboInOwl:hasDbXref "PMID:17483847") obo:IAO_0000115 obo:CL_4040002 "Glial cell that provides support to the enteric nervous system. It is involved in enteric neurotransmission, in maintaining the integrity of the mucosal barrier of the gut and serves as a link between the nervous and immune systems of the gut.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040002 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040002 "2023-04-03T15:15:40Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25170211") oboInOwl:hasExactSynonym obo:CL_4040002 "enteric glial cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11169131") oboInOwl:hasExactSynonym obo:CL_4040002 "enteroglial cell")
+AnnotationAssertion(rdfs:label obo:CL_4040002 "enteroglial cell")
+EquivalentClasses(obo:CL_4040002 ObjectIntersectionOf(obo:CL_0000125 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002005)))
```

Comment:
Agree with this equivalence axiom, but due to an upper level axiom, this cell type is being inferred as a 'Schwann cell'.

<img width="466" alt="Screenshot 2023-04-06 at 09 50 13" src="https://user-images.githubusercontent.com/95643797/230325592-c5347469-2362-4e29-8dee-dface3c65d27.png">

I do not think this is accurate. Several statements in the references and the examples below suggest that enteric glial cells are distinct from Schwann cells.

PMID:11169131:
"In contrast to myelinating and nonmyelinating Schwann cells [enteric glial cells] do not express Ran-1 in rats8 or Schwann cell myelin protein in avians."

https://doi.org/10.1073/pnas.88.14.6358:
"We produced earlier a monoclonal antibody against Schwann cell myelin protein (SMP), a glycoprotein expressed on Schwann cells (SC) but not on satellite cells of the ganglia or enteric glial cells."

This classification is due to the equivalence axiom of 'Schwann cell' being too broad:
'glial cell' and ('part of' some 'peripheral nervous system')

Although this is outside the scope of this ticket, it impacts this new term. I'll plan to create a separate ticket to address this.

Reply:
@bvarner-ebi Should I keep the Equivalent To as is and a ticket will be create to solve the equivalence axiom of 'Schwann cell' being too broad?

Reply:
I created #1926.
Yes, I think the Equivalent To is accurate and the inferred subclass will be resolved once #1926 is merged.

Reply:
FYI, #1926 has been merged and appears to resolve this issue.

Reply:
@bvarner-ebi thanks

Reply:
I merged the updated master into this branch, so the update can be seen that `subClass of some 'Schwann cell'` is no longer inferred.